woocommerce/woocommerce-gateway-stripe
# Add spacing after UPE container on checkout.

# Changes proposed in this Pull Request:

Issue: Link to the GitHub issue this PR addresses (if appropriate).
Fixes #1976 

In some instances, the "save payment information" checkbox could touch or overlap with input elements from the UPE iframe. This adds 4px of bottom margin to the iframe container to prevent this.


# Testing instructions

1. Apply PR.
2. Ensure that UPE is enabled.
3. Add item to cart.
4. Go to checkout screen.
5. Select the iframe inside of `#wc-stripe-upe-element` in the `Elements` panel of dev tools to highlight the container on the screen.
6. Ensure that there is no overlap of that highlighted element and the "Save Payment Information" checkbox.

Before (with Twenty Twenty-One Theme):
<img width="613" alt="134962263-85576840-ae5c-4e55-beff-213dfdb6147d" src="https://user-images.githubusercontent.com/1558827/135282423-40429736-aeb6-4933-8eb4-e9459f9ed755.png">

After (with Twenty Twenty-One Theme):
<img width="605" alt="134963075-3b4e67e8-bb60-44bd-9431-9ea0b18910c8" src="https://user-images.githubusercontent.com/1558827/135282483-636ef776-94b3-4232-9b60-5cf26103d487.png">
